entrypoint.sh

```diff
@@ -455,7 +455,7 @@ function rerun_lint() {
 }
 
 function initiate_code_review() {
-  if [ -n "$GITHUB_ACTIONS" ]; then
+  if [[ -n ${GITHUB_ACTIONS:+x} ]]; then
```

Comment:
This is the primary fix; everything else are updates which `task update` doesn't handle automatically.